monai/losses/focal_loss.py

```
@@ -70,19 +70,21 @@ def __init__(
         include_background: bool = True,
         to_onehot_y: bool = False,
         gamma: float = 2.0,
-        alpha: float | None = None,
+        alpha: float | Sequence[float] | None = None,
```

Ruff (0.14.8)

Undefined name `Sequence`

(F821)